integration-tests/utils/integration-utils.js

```diff
@@ -169,10 +169,15 @@ const dismissNotifications = async page => {
   return !!notificationCloseButtons.length && delay(1000) // delayed for alerts to animate off
 }
 
-const signIntoTerra = async (page, token) => {
-  await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true })
+const signIntoTerra = async (page, { token, testUrl }) => {
+  if (testUrl) {
+    await page.goto(testUrl, { waitUntil: ['load', 'domcontentloaded', 'networkidle0'], timeout: 60 * 1000 })
+  }
+  await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true, timeout: 60 * 1000 })
+  await waitForNoSpinners(page)
+  await page.evaluate(token => window.forceSignIn(token), token)
```

Comment:
Could you explain why the line above was added since it doesn't seem to exist in the current version of `dev`?

Reply:
I see now. Thanks.